android/java/org/chromium/chrome/browser/crypto_wallet/activities/AssetDetailActivity.java

```diff
@@ -132,7 +157,51 @@ protected void triggerLayoutInflation() {
 
         TextView assetTitleText = findViewById(R.id.asset_title_text);
         assetTitleText.setText(mAssetName);
-        if (!mAssetLogo.isEmpty()) {
+        mBtnBuy = findViewById(R.id.btn_buy);
+        Button btnSend = findViewById(R.id.btn_send);
+        mBtnSwap = findViewById(R.id.btn_swap);
+
+        if (mCoinMarket) {
+            TextView informationLabel = findViewById(R.id.information);
+            CardView coinMarketInfo = findViewById(R.id.card_view_coin_market_info);
+            AndroidUtils.show(informationLabel, coinMarketInfo);
+
+            if (mMarketCapRank != -1) {
+                TextView rank = findViewById(R.id.rank);
+                rank.setText(String.valueOf(mMarketCapRank));
+            }
+
+            if (mVolume24Hour != -1) {
+                TextView volume = findViewById(R.id.volume);
+                volume.setText(mMarketModel.getFormattedUsdBillions(mVolume24Hour));
+            }
+
+            if (mMarketCap != -1) {
+                TextView marketCap = findViewById(R.id.market_cap);
+                marketCap.setText(mMarketModel.getFormattedUsdBillions(mMarketCap));
+            }
+
+            DisplayMetrics displayMetrics =
+                    ContextUtils.getApplicationContext().getResources().getDisplayMetrics();
+            final int sizePx = dpToPx(displayMetrics, ASSET_LOGO_SIZE_DP);
+            ImageLoader.downloadImage(mAssetLogo, Glide.with(this), true,
```

Comment:
Setting a drawable for a `textView` using our `ImageLoader` class required some trickery as we didn't provide a method that would accept a `textView`. 
Solution: Implemented a new method that accepts a more flexible `CustomTarget` where we can specify the custom action.

```java
downloadImage(String url, final RequestManager requestManager,
            final boolean isCircular, final int roundedCorners,
            final CustomTarget<Drawable> customTarget, final Callback callback)
```